src/core/mac/mac_types.hpp

```diff
@@ -421,6 +421,24 @@ struct Addresses
     Address mDestination; ///< Destination address.
 };
 
+/**
+ * This structure represents two PAN IDs corresponding to source and destination also indicating whether or not each is
+ * present.
+ *
+ */
+struct PanIds
+{
+    void SetSource(PanId aPanId) { mHasSource = true, mSource = aPanId; }
+    void SetDestination(PanId aPanId) { mHasDestination = true, mDestination = aPanId; }
+    void SetToNoSoruce(void) { mHasSource = false; }
```

Comment:
```suggestion
    void SetToNoSource(void) { mHasSource = false; }
```

Reply:
Thanks. I actually intended to remove all of these methods since we can already track if we have src/dst PAN ID based on presence of the src/dst address. So we can change the `PanIds` to be just two PanId values without the `mHas{}` properties.

Will change in next push.